bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/TemplateUtils.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.beam.sdk.options.ValueProvider;
 import org.apache.hadoop.hbase.client.Scan;
 import org.apache.hadoop.hbase.filter.ParseFilter;
+//import static com.google.cloud.bigtable.hbase.BigtableOptionsFactory;
```

Comment:
I'd like to import the constants directly but there is some project interdependency in here which I don't understand.

Reply:
Except for the `static` keyword this should work, please use the constant.

I'm guessing your IDE is complaining  about the missing symbol. To workaround the issue run `mvn clean install` in the root of the project, then if you are using intellij disable the `with-shaded` profile in the maven tool window. If using eclipse, close the `bigtable-hbase-1.x-shaded` module.